### DISCUSSION TITLE
GitHub Copilot    Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by 271400035** [community/community#113106] [GitHub Education] [answered]
### Hi everyone,

I was trying GitHub Copilot for a month first, and during the trial I did student authentication, but now that the month has expired, GitHub Copilot is telling me that I can't use it anymore, and clicking on the Open button won't redirect me to the Open page. In addition, my GitHub Copilot but shows that it is letting me open as shown in the picture below, but inside the billing it shows that I am eligible to use it for free, how can I solve this problem?

![image](https://github.com/community/community/assets/21153934/ece03399-9d79-4000-8444-15ffeb5e0b63)
![image](https://github.com/community/community/assets/21153934/98e818ed-eecd-4d3e-9efa-008337069ddf)

**Comment by RooobinYe** [comment]
same.

**Comment by BalteMD** [comment]
same

**Comment by huangjiachenge** [comment]
same.

**Comment by huangjiachenge** [comment]
This is the error log in my vsCode

2024-03-19 10:14:10.130 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-19 10:14:10.176 [error] [default] Extension activation failed: "Contact Support. You are currently logged in as huangjiachenge."

**Comment by hogivano** [comment]
same

**Reply by 271400035** [reply, marked as answer]
My issue has been successfully resolved, if you have a similar issue the final solution is as follows, find github support to submit a work order using the following description:

Hello, I am now student certified, please cancel my 30 day trial and switch to Student-2024 coupon.

**Comment by tuankiet-0405** [comment]
same
